src/mock_network.zig

```diff
@@ -0,0 +1,229 @@
+const std = @import("std");
```

Comment:
This was later moved to `test/packet_simulator.zig` and changed a fair bit. I think this may still be an old `beta2` artifact creeping in?